docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-model/calibrating-sensors/lidar-imu-calibration/index.md

```diff
@@ -0,0 +1,263 @@
+  (you can only select `sensor_msgs/PointCloud2` and `sensor_msgs/Imu` messages),
+  and convert your split ROS 2 bag file to ROS 1 bag.
+
+Create a yaml file with name `out.yaml` which contains your lidar and imu topics:
+
+```yaml
+output_bags:
+  - uri: splitted_bag
+    topics: [/your/imu/topic, /your/pointcloud/topic]
+```
+
+Split your ROS 2 bag file:
+
+```sh
+ros2 bag convert -i <YOUR-ROS2-BAG-FOLDER> -o out.yaml
+```
+
+Convert your split ROS 2 bag file to ROS 1 bag file:
+
+```sh
+# install bag converter tool (https://gitlab.com/ternaris/rosbags)
+pip3 install rosbags
+
+# convert bag
+rosbags-convert <YOUR-SPLITTED-ROS2-BAG-FOLDER> --dst <OUTPUT-BAG-FILE>
+```
+
+## Lidar-Imu Calibration
+
+As a first step, we need to install docker on our system. You can install docker
+using [this link](https://docs.docker.com/engine/install/ubuntu/), or you can use
+the following commands to install docker using the Apt repository.
+
+Set up Docker's Apt repository:
+
+```sh
+# Add Docker's official GPG key:
+sudo apt-get update
+sudo apt-get install ca-certificates curl gnupg
+sudo install -m 0755 -d /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+sudo chmod a+r /etc/apt/keyrings/docker.gpg
+
+# Add the repository to Apt sources:
+echo \
+  "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  "$(. /etc/os-release && echo "$VERSION_CODENAME")" stable" | \
+  sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt-get update
+```
+
+Install the Docker packages:
+
+```sh
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+```
+
+To check if docker is installed correctly, you can run the following command:
+
+```sh
+sudo docker run hello-world
+```
+
+Before finishing the installation, we need to add our user to the docker group.
+This will allow us to run docker commands without sudo:
+
+```sh
+sudo groupadd docker
+sudo usermod -aG docker $USER
+```
+
+!!! warning
+
+    After running the above command, you need to logout and login again to be able
+    to run docker commands without sudo.
+
+After installing docker, we are ready to run the calibration tool. As a first step,
+you should clone the calibration repository:
+
+```sh
+git clone https://github.com/leo-drive/OA-LICalib.git
+```
+
+Then, you need to build the docker image:
+
+```sh
+cd OA-LICalib/docker
+sudo docker build -t oalicalib .
+```
+
+After building the docker image, you need to create a container from the image:
+
+!!! warning
+
+    You need to update REPO_PATH with the path to the cloned repository
+    on your system.
+
+```sh
+export REPO_PATH="/path/to/OA-LICalib"
+docker run -it --env="DISPLAY" --volume="$HOME/.Xauthority:/root/.Xauthority:rw" --volume="/tmp/.X11-unix:/tmp/.X11-unix:rw" --volume="$REPO_PATH:/root/calib_ws/src/OA-LICalib" calib:v1 bash
```

Comment:
Thanks for this great work! I tried to use tool. While executing this piece of command, it gives `Unable to find image 'calib:v1' locally`. I guess docker image's name is oalicalib now, when i tried with oalicalib instead of calib:v1 it worked for me

Reply:
One more issue I had encountered is, I noticed that the workspace directory has changed to catkin_oa_calib. --volume="$REPO_PATH:/root/calib_ws/src/OA-LICalib" should change I think